examples/Resource Bundle Example/Example Pods/Images.xcassets/Contents.json

```diff
@@ -0,0 +1,6 @@
+{
+  "info" : {
+    "version" : 1,
+    "author" : "xcode"
+  }
+}
```

Comment:
Sure I can do that. Xcode (7.2) generated these though, so if someone goes and modifies this later, you'll run into the same issue. Would you still like this fixed?